flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/source/ScanContext.java

```diff
@@ -72,6 +72,10 @@ class ScanContext implements Serializable {
   private static final ConfigOption<Boolean> INCLUDE_COLUMN_STATS =
       ConfigOptions.key("include-column-stats").booleanType().defaultValue(false);
 
+  private static final ConfigOption<Integer> MAX_SNAPSHOTS_PER_MONITOR_INTERVAL =
+      ConfigOptions.key("max-snapshot-per-monitor-interval").intType()
```

Comment:
it will be more clear if the config is defined as `max-snapshot-count-per-monitor-interval` or `max-snapshot-count-per-incremental-scan`